libs/traits/src/lib.rs

```diff
@@ -187,6 +187,18 @@ pub trait PoolReserve<AccountId, CurrencyId>: PoolInspect<AccountId, CurrencyId>
 	fn deposit(pool_id: Self::PoolId, from: AccountId, amount: Self::Balance) -> DispatchResult;
 }
 
+/// A trait that supports modifications of pool write-off policies
+pub trait PoolWriteOffPolicyMutate<PoolId> {
+	type Rate;
+	type WriteOffRule: Encode + Decode + Clone + TypeInfo + Debug + PartialEq;
```

Comment:
Although for these bounds I think it doesn't matter. Personally, I prefer to leave the associated types as generic as possible, adding here only the bounds required by the trait itself. Later it will ease the mocking/testing part and reason about what's really needed for the implementer.